src/styles/components/_card-container.scss

```diff
@@ -0,0 +1,20 @@
+.card-container {
+  display: flex;
+  justify-content: space-between;
```

Comment:
More correctly use justify-content: center, because space-between can set additional indents and they are already set with

Reply:
agreed, but than `flex-start` looks better.